DayrSuryan/DSEthiop2.xml

```diff
@@ -159,7 +159,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                     <origDate notBefore="1850" notAfter="1949">Second half of the nineteenth/early twentieth century</origDate>.
 
                     The main Text I (Gospel of John) of <ref target="#p2">Unit 1</ref> was written by <persName ref="PRS11986GabraI" role="scribe">Gabra ʾIyasus</persName>,
-                     possibly before <date>1921/22</date> (cp. <ref target="#a1">Addition 1 of Unit 1</ref>).
+                     possibly before <date>1912/1913</date> (cp. <ref target="#a1">Addition 2 of Unit 1</ref>).
```

Comment:
if it is Addition 2, then it should also be changed to a2

Reply:
ok